mailer.rst

```diff
@@ -55,14 +55,10 @@ over SMTP by configuring the DSN in your ``.env`` file (the ``user``,
 
         // config/packages/mailer.php
         use function Symfony\Component\DependencyInjection\Loader\Configurator\env;
-        use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
-
-        return static function (ContainerConfigurator $containerConfigurator): void {
-            $containerConfigurator->extension('framework', [
-                'mailer' => [
-                    'dsn' => env('MAILER_DSN'),
-                ],
-            ]);
+        use Symfony\Config\FrameworkConfig;
+
+        return static function (FrameworkConfig $frameworkConfig): void {
+            $frameworkConfig->mailer()->dsn(env('MAILER_DSN')->string());
```

Comment:
what does the `string()` at the end?
sorry I rarely use such config way thus only bare comparing array config vs php function here

Reply:
It's called an "[Environment Variable Processor](https://symfony.com/doc/current/configuration/env_var_processors.html#built-in-environment-variable-processors)"

Reply:
ho i see now, know it in other config lang but not php, thx for clarifying 👍🏻